shapes/neurosciencegraph/datashapes/atlas/imagestack/schema.json

```diff
@@ -87,29 +86,35 @@
     {
       "@id": "this:SliceResolutionShape",
       "@type": "sh:NodeShape",
-      "and": [
+      "property": [
         {
-          "node": "https://neuroshapes.org/commons/quantitativevalue/shapes/QuantitativeValueShape"
+          "path": "nsg:widthResolution",
```

Comment:
I would suggest to have every resolution to a quantitative value even if the unit code is the same between widthResolution and heightResolution.

Reply:
I agree! Will make those changes.